packages/features/bookings/components/PeopleFilter.tsx

```diff
@@ -21,7 +21,7 @@ export const PeopleFilter = () => {
   const { data: query, pushItemToKey, removeItemByKeyAndValue, removeAllQueryParams } = useFilterQuery();
   const [searchText, setSearchText] = useState("");
 
-  const members = trpc.viewer.teams.listMembers.useQuery({});
+  const members = trpc.viewer.teams.legacyListMembers.useQuery({});
```

Comment:
I still need `legacyListMembers` because we need unique members from all the teams making sure a members of multiple team doesn't show up more than once but `cursor` and `distinct` property wasn't working properly

```

  const teamMembers = await prisma.membership.findMany({
    where: whereCondition,
    select: {
      id: true,
      role: true,
      accepted: true,
      teamId: true,
      user: { select: userSelect },
    },
    cursor: cursor ? { id: cursor } : undefined,
    take: limit + 1,
    orderBy: { id: "asc" },
    distinct: ["userId"],
  });
  ```